themes/rad/layouts/index.html

```diff
@@ -4,25 +4,25 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Open source development is a peer&ndash;to&ndash;peer process.
+        With Radicle you can collaborate directly, without giving your
+        data to third parties or setting up your own server.
       </p>
     </div>
     <div id="term">
       <h2>Terminal-first</h2>
       <p>
-        A Radicle project contains a git repository as well as the
-        issues and proposals associated with that repo. All accessible from
-        your terminal.
+        A Radicle project contains a git repository, plus the associated
+        issues and proposals. Access all the conversations
+        around your codebase from the comfort of your command line.
       </p>
     </div>
     <div id="prog">
       <h2>Programmable</h2>
       <p>
-        Ever wanted to tweak your code collaboration service? With Radicle each 
-        unit of functionality &ndash;<em>a machine</em>&ndash; is its
-        own litte P2P program, written in the Radicle language. You can change
-        them to suit your needs, or create entirely new ones. More on this soon.
+        Each Radicle utility is a little P2P program we call a Radicle <em>machine</em>.
+        Change them to suit your needs, or create entirely new ones.
+        Developer resources coming soon!
```

Comment:
Also like this one.

Reply:
I still feel that with both of them we haven't nailed the fact that this gives you the power to write your own issues app etc. We talk about these programs, but not what they are good for, or why you'd want to build them. Also I think we need to drop P2P here too. We've said it twice already by now.